src/Card/CardActionIcon.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { IconButton, Icon } from '..';
+import { Close, MoreVert } from '../../icons';
+
+const CardActionIcon = React.forwardRef(
+  ({
+    className,
+    actionIcon,
+    onClick,
+    isActive,
+    variant,
+  }, ref) => {
+    if (actionIcon === 'overflow') {
```

Comment:
Hmm, I feel `CardActionIcon` could probably be implemented in a bit more flexible way, that would allow consumers to customize it for any `IconButton` icon for the intended use case.

I'm thinking we could treat `CardActionIcon` essentially as a pass-thru component to `IconButton`, but with the extra styles to correctly position it in the `Card`, where `Card.ActionIcon` would take pass (?) of its props down to the underlying `IconButton`, perhaps with restrictions on which variants are valid for `Card.ActionIcon`.

With `Card.ActionIcon` as a sub-component, in theory, it could then be wrapped / composed by `OverlayTrigger` to optionally add a tooltip/popover to it., if needed.

```jsx
<Card>
  <Card.ActionIcon
    src={Close}
    alt="Close"
    variant="dark"
  />
</Card>
```

Just a thought. Curious to hear your thoughts on how to make this a bit more flexible to support more use cases (e.g., using different icons, changing the `IconButton` variant, opting in to using a `ModalPopup` or `Tooltip` alongside the `Card.ActionIcon`, etc.).